lib/msal-browser/src/crypto/CryptoOps.ts

```diff
@@ -198,4 +212,22 @@ export class CryptoOps implements ICrypto {
         const hashBytes = new Uint8Array(hashBuffer);
         return this.b64Encode.urlEncodeArr(hashBytes);
     }
+
+    /**
+     * Returns the public key from an asymmetric key pair stored in IndexedDB based on the
+     * public key thumbprint parameter
+     * @param keyThumbprint 
+     * @returns Public Key JWK string
+     */
+    async getAsymmetricPublicKey(keyThumbprint: string): Promise<string> {
+        const cachedKeyPair = await this.cache.asymmetricKeys.getItem(keyThumbprint);
+
+        if (!cachedKeyPair) {
+            throw BrowserAuthError.createSigningKeyNotFoundInStorageError();
+        }
+
+        // Get public key as JWK
+        const publicKeyJwk = await this.browserCrypto.exportJwk(cachedKeyPair.publicKey);
+        return BrowserCrypto.getJwkString(publicKeyJwk);
+    }
```

Comment:
Would be great to see some more log messages in this class.

Reply:
Added some more logging, trying to balance verbose logging with not having redundant logging since the APIs that call these APIs do their own logging around the calls for protocol-specific logging.